staging/src/k8s.io/apiextensions-apiserver/test/integration/fixtures/resources.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package testserver
+package fixtures
```

Comment:
I think this file needs a 2017 -> 2018 bump too.

Reply:
It's just renamed. We don't change the year usually for that.